docs/content/en/docs/IOP/Deployment/Kubernetes/_index.md

```diff
@@ -83,10 +83,10 @@ kubectl create secret tls <keypair> --key=tls.key --cert=tls.crt
 Once all this is done, we can carry on with the deployment by running:
 
 ```bash
-helm install iop sciencemesh/iop \
-  --set-file revad.configFiles.revad\\.toml=standalone.toml \
-  --set-file revad.configFiles.users\\.json=users-cern.json \
-  --set-file revad.configFiles.ocm-providers\\.json=providers.demo.json \
+helm upgrade -i iop sciencemesh/iop \
```

Comment:
Would it also work if it's not already installed?

Reply:
Yep,  also the `upgrade -i` formula is actually starting to get community traction since it covers both upgrade and install as fallback. From the [`helm upgrade` docs](https://helm.sh/docs/helm/helm_upgrade/):

```
  -i, --install                      if a release by this name doesn't already exist, run an install
```